packages/core/src/router/urlParams.factory.ts

```diff
@@ -1,25 +1,21 @@
 import { ParametricRegExp } from './router.interface';
+import * as pathToRegexp from 'path-to-regexp';
 
 export const factorizeRegExpWithParams = (path: string): ParametricRegExp => {
-  const pathParameters = /:([^\/]+)/g;
-  const parameters: string[] = [];
+  const keys: pathToRegexp.Key[] = [];
+  const preparedPath = path
+    .replace(/\/\*/g, '/(.*)') /* Transfer wildcards */
```

Comment:
What's this line for?

Reply:
By default `path-to-regexpt` doesn't support asterisk wildcard parameters `*`, instead of it it handles `(.*)` notation. In general this replacement should math paths like:
- `/foo/*/bar`
- `/foo/*`